pkg/controller/replication/replication_controller.go

```diff
@@ -478,10 +573,19 @@ func (rm *ReplicationManager) manageReplicas(filteredPods []*api.Pod, rc *api.Re
 		var wg sync.WaitGroup
 		wg.Add(diff)
 		glog.V(2).Infof("Too few %q/%q replicas, need %d, creating %d", rc.Namespace, rc.Name, rc.Spec.Replicas, diff)
+		// needs to set the gvk to generate ownerReference
```

Comment:
Ah. acronyms in comments/docstrings tend to confuse, unless it's something we officially expose through the api. There is no local context on what group version or kind is anywhere in this file, so it's also worth mentioning *why* we need to set the gvk to generate ownerReference instead of that we are setting it.